docs/documentation/actions/micrometers.md

```diff
@@ -0,0 +1,285 @@
+    |    *     | `name`       | String                           | Counter name. It's required if counter input is null.    |
+    |    *     | `counter`    | [Counter](https://javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/Counter.html){:target=_blank}             | Counter instance. It's required if name input is null. |
+    |          | `description`| String                           | Counter  description                                 |
+    |          | `unit`       | Map <String, String\>            | Count [unit](https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/BaseUnits.java){:target=_blank}                                                 |
+    |          | `tags`       | List<String\>                    | key,value list representing tags. A tag is a Key/value pair representing a dimension of a meter used to classify and drill into measurements.                                  |
+    |          | `registry`   | [MeterRegistry](https://javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/MeterRegistry.html){:target=_blank}             | Creates and manages your application's set of meters  |
+    |     *    | `increment`  | String                           | Positive number by which the counter will be incremented. |
+
+=== "Outputs"
+
+    |    Name               | Type                                                                                                                               | Description             |
+    |:----------------------|:-----------------------------------------------------------------------------------------------------------------------------------|:------------------------|
+    | `micrometerCounter`   | [Counter](https://javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/Counter.html){:target=_blank}  | The incremented counter |
+
+### Example
+
+=== "Kotlin"
+
+``` kotlin
+MicrometerCounterAction(
+    name = "products_likes",
+    description = "products likes counter",
+    tags = listOf(
+        "product", "1",
+        "liked_feature", "color"
+    ),
+    increment = "1"
+)
+```
+
+# Gauge
+
+Use this action to report a [gauge](https://micrometer.io/docs/concepts#_gauges){:target=_blank} metric.
+
+=== "Inputs"
+
+    | Required | Name              | Type          |  Description                                               |
+    |:--------:|:------------------|:--------------|:-----------------------------------------------------------|
+    |    *     | `name`            | String        | Gauge name. It's required if counter input is null.        |
+    |          | `description`     | String        | Gauge  description                                         |
+    |    *     | `gaugeObject`     | Object        | Gauge will return the current value of this object .       |
```

Comment:
Instead of * for required, i would write : "on condition" or "complex condition"